conf/ceh.config

```diff
@@ -0,0 +1,31 @@
+//Profile config names for nf-core/configs
+ params {
+            config_profile_description = 'Center for Evolutionary Hologenomics & Section for Evolutionary Genomics @ GLOBE - CEH profile provided by nf-core/configs.'
```

Comment:
```suggestion
            config_profile_description = 'Center for Evolutionary Hologenomics @ GLOBE - CEH profile provided by nf-core/configs.'
```

In the future if other groups start their own private cluster it would make sense to have these as different internal profiles, but until then just make it for all CEH.

Reply:
You could also maybe add University of Copenhagen, GLOBE isn't so well known I guess

Reply:
All of us who work for Tom are all part of the Section for Evolutionary Genomics and many are also part of the Center for Hologenomics. I am not part of the latter and many who work in aDNA in our group are not part of the CEH either, but some are.

Reply:
I've change dit all to seg_globe, in case the CEH gets their own servers in the future